_targets.R

```diff
@@ -0,0 +1,42 @@
+library(targets)
+source("code.R")
```

Comment:
This line should be sourcing any of your `.R` files that contain functions needed by your targets. I think with how you have your repo setup, this will need to include 3 lines - `source("1_fetch/src/fetch_data.R")`, `source("2_process/src/process_data.R")`, `source("3_visualize/src/plot_data.R")`.